### DISCUSSION TITLE
Making a Discord Server

### DISCUSSION THREAD

**Discussion by brotherelric** [discord-py-ui/discord-ui#107] [Ideas]
I think it would be nice for this repository to have a Discord server for future announcements, [commit notifications](https://cb-ashik.hashnode.dev/how-to-get-github-notifications-in-discord-server), discussions and help.

**Comment by kvsxxx** [comment]
a discord server ([invite here](https://discord.gg/bDJCGD994p)) exists, the link was at the end of the readme
